python/gui/MaaS/static/maas/js/ready_evaluation.js

```diff
@@ -137,8 +137,6 @@ async function submit_evaluation(event) {
         editorData.view.setOption("readonly", true);
     }
 
-    $(event.target).button("disable");
```

Comment:
Was this accidentally committed?

Reply:
No, it's a fix for annoying behavior I encountered while working. You can submit an evaluation, but the button becomes permanently disabled. I removed the behavior. It will allow you to rapid fire identical evaluations (not ideal), but if something fails and you only need to change a part of the config, you no longer need to refresh the page.